src/js/utils/__tests__/RouterUtil-test.js

```diff
@@ -208,4 +208,33 @@ describe("RouterUtil", function() {
       expect(RouterUtil.getQueryStringInUrl()).toEqual(expectedResult);
     });
   });
+
+  describe("#replaceQueryInPathString", function() {
```

Comment:
Or keep the method as it is, and add a test that shows what happens when calling on a url with a key with no value like:
`http://localhost:4200/#/catalog/packages/arangodb3?version=` (expect no-op)